components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1589 @@
+$$ (continuous-momentum2)
+
+with total forces given by:
+
+$$
+\mathbf{F}_\text{total}[V(t)] =
+- \int_{V(t)} \rho\, \mathbf{f} \times \mathbf{u} \, dV
+- \int_{V(t)} \rho\, \nabla_{3D} \Phi \, dV
+- \int_{\partial V(t)} p \, \mathbf{n} \, dA
++ \int_{\partial V(t)} \boldsymbol{\tau} \cdot \mathbf{n} \, dA
+$$ (momentum-Ftotal)
+
+Each term on the right-hand side corresponds to a physically distinct force acting on the fluid within the control volume:
+
+- The first term is the **Coriolis force**, where $ \mathbf{f} $ is the vector Coriolis parameter (e.g., $ f \hat{\mathbf{z}} $ on the sphere).
+- The second term represents the **gravitational force**, expressed in terms of the gradient of the gravitational potential $ \Phi(x, y, z, t) $, which may include effects such as tides and self-attraction and loading.
+- The third term is the **pressure force**, which acts on the boundary surfaces and is naturally expressed as a surface integral. It gives rise to both horizontal pressure gradients and contributions from sloping surfaces.
+- The fourth term represents the surface stresses. They include **horizontal stress forces** across the vertical sides of the control volume, such as those due to lateral friction or subgrid momentum transfer.
+They also include **vertical stress forces** (e.g., wind stress at the ocean surface and bottom drag at the seafloor), projected into horizontal momentum and acting across the top and bottom surfaces.
+
+#### Pressure Term
+
+The pressure force term may be converted from the boundary to the interior with Gauss' divergence theorem (see [Kundu et al. 2016](https://www.amazon.com/dp/012405935X/) p. 119),
+
+$$
+- \int_{\partial V(t)} p \, \mathbf{n} \, dA 
+= - \int_{V(t)} \nabla_{3D} p \, dV .
+$$ (gradp-Gauss)
+
+Considering only the horizontal components, we have
+
+$$
+ - \int_{V(t)} \nabla_{\perp} p \, dV
+= - \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} 
+  \nabla_{\perp} p \, dz \, dA
+= - \int_{A}   \overline{\nabla_{\perp} p}^{z}  \, dA.
+$$ (gradp-h)
+
+#### Stress Term
+
+Likewise, the stress tensor integrated over the surface may be converted to a volume integral with Gauss' theorem ([Kundu et al. 2016](https://www.amazon.com/dp/012405935X/) p. 125 eqn 4.20b),
+
+$$
+\int_{\partial V(t)} \boldsymbol{\tau} \cdot \mathbf{n} \, dA
+= \int_{V(t)} \nabla_{3D} \cdot \boldsymbol{\tau} \, dV
+$$ (stress-Gauss)
+
+for clarity, this can be written in index notation as
+
+$$
+\int_{\partial V(t)} n_i \tau_{ij} \, dA
+= \int_{V(t)} \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) \, dV
+$$ (stress-Gauss-index)
+
+Taking only the horizontal ($j$=1,2),
+
+$$
+ \int_{V(t)} \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) \, dV
+= \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} 
+ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) 
+\, dz \, dA
+=  \int_{A} 
+  \overline{
+ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) }^z dA
+$$ (gradp-h)
+
+#### Horizontal momentum
```

Comment:
The top/bot vs. k and k+1 notation in this section is messy and needs a little cleaning up.  Let's stick with `top`, k and k+1 to be consistent with the previous notation change.